onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -1397,6 +1397,23 @@ TEST(InferenceSessionTests, TestParallelExecutionWithCudaProvider) {
   ASSERT_TRUE(!status.IsOK());
 }
 
+TEST(InferenceSessionTests, TestSSDMobileNet) {
```

Comment:
Possibly there exists a better way to cover this change without testing it at an InferenceSession level and without having to checking-in a 28 MB model.

(For now keeping this here as a "test" placeholder)

Reply:
There's an existing test where a graph is manually created with an initializer at outer scope. Perhaps it could be used as a base. ConstantFoldingSubgraph in onnxruntime\test\optimizer\graph_transform_test.cc

---
In reply to: [303690070](https://github.com/microsoft/onnxruntime/pull/1413#discussion_r303690070) [](ancestors = 303690070)

Reply:
Thanks, will create a test based on that one.